sdk/storage/inc/common/reliable_stream.hpp

```diff
@@ -26,8 +26,6 @@ namespace Azure { namespace Storage {
   {
     // configures the maximun retries to be done.
     int64_t MaxRetryRequests;
-    // Use for testing purposes only.
-    bool DoInjectError;
```

Comment:
well, this is expected to be used on unit testing to simulate a fail while reading from the stream.
If we remove it, we would need to come up with some other way to test a Read() fail for unit testing

see the GO example:
https://github.com/Azure/azure-storage-blob-go/blob/master/azblob/zc_retry_reader.go#L120

Reply:
I did target test for this scenario. It works.

Reply:
and I don't feel like adding code just for test purpose into production codebase